docs/proposals/mcs/remove-derived-mcs.md

```diff
@@ -0,0 +1,352 @@
+
+## Motivation
+
+Currently, in different cluster members of Karmada, if you need to access the same service across clusters, you need to use different ip or domain names with a "derived-" prefix to access. This will increase the complexity of the user's use. We hope that users can use the same domain name to achieve cross-cluster access.
+
+### Goals
+
+* Keep the original access method. Avoid the need to make modifications to the customer's system after the upgrade.
+* Added the use of a unified domain name for cross-cluster access. The domain name is the access domain name of the cluster where the service resides.
+
+### Non-Goals
+
+none
+
+## Proposal
+
+### User Stories
+
+According to the current Karmada's  [mcs configuration](https://karmada.io/docs/userguide/service/multi-cluster-service) , if the service is currently in member1, the user needs to access the service in member1, needs to use the domain name: "serve.default.svc.cluster.local", and the domain name in member2 will be: "derived-serve.default.svc.cluster.local". As a result, when customers use the service, they need to add the prefix "derived-" to access the domain name of the service. This brings additional inconvenience. So I hope to be able to:
+
+1. Provide a unified access method across clusters.
+
+2. The original access method remains unchanged. So that existing business can also work normally.
+
+### Notes/Constraints/Caveats (Optional)
+
+1. The coredns version must be v1.9.3 or above, and the matching k8s verison is at v1.21.0 or above, otherwise the multicluster feature is not supported
+
+### Risks and Mitigations
+
+
+
+## Design Details
+
+### API change
+
+none
+
+### Components change
+
+Here is give the implementation plan:`ServiceImport ` + karmada.
+
+#### 1) Use ServiceImport to generate rr records
+
+This solution is mainly realized by combining the `multicluster` plug-in of `corends`. It is divided into two parts: `coreDNS` transformation and `karmada.` For the ``coreDNS` `, here is two implementation plan.
+
+##### (1) coreDNS+multicluster solution
+
+This solution is mainly realized by the `multicluster` plug-in of `corends`.
+
+###### **Principle Description**
+
+In order to adapt to mcs, `coreDNS` launched the `multicluster` plug-in. After the plug-in completes the configuration and deployment, it will parse `ServiceImport`. The main content of the analysis is the name, namespace and ips, and an rr record is generated accordingly. Then client pod can cross-cluster access services.
+
+The workflow of the `multicluster` plugin is as follows:
+
+![diagrammatic drawing of remove mcs](./remove-derived-mcs.png)
+
+The diagram is explained below:
+
+* Here give a service named `foo` , and deployed in `member1`. The full domain name that needs to be used to access the `foo` service in member1 is `foo.default.svc.cluster.local`.
```

Comment:
> This explanation in the latter half of the sentence doesn't seem to have a direct connection with the proposal we put forward.

I see, how about instead of these:
"Here give a service named `foo` in `member1`. We should use the full domain name: `foo.default.svc.cluster.local` to access this service. But we cannot use the same domain name in `member2`."